### DISCUSSION TITLE
Which parts exactly are needed for v2?

### DISCUSSION THREAD

**Discussion by owenashurst** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#547] [Q&A] [answered]
Hi there,

I've just come across this project and I just ordered some parts already such as the capacitors and resistors that apparently I needed according to when I uploaded the PCB files. However, in the README in the root, it specifies that you need the ESP8266 NodeMCU 1.0 which from other photos I've seen of the V2 board it specifies v3? 

Also, which 8 channel bidirectional level converter do I need exactly? I've found one on eBay here, but are the pins right?

https://www.ebay.co.uk/itm/201637192817?mkcid=16&mkevt=1&mkrid=711-127632-2357-0&ssspo=fVLSjdfIQRm&sssrc=4429486&ssuid=&var=&widget_ver=artemis&media=COPY

I'm in the UK so any UK links would be appreciated.

Thanks for an awesome project!
Owen

**Comment by visualapproach** [comment, marked as answer]
Hi! NodeMCU v2 or 3 is both okay on pcb_v2/pcb_v2b. It's just different pin spacing.
Caps and resistors form a signal filter which may or may not be needed.
I'd suggest "the red" level shifter. It seems to be more successful but as with the filters you may have to experiment a little if you have a picky pump. Search for 8 channel bi directional level shifter on eBay.

**Reply by owenashurst** [reply]
> Hi! NodeMCU v2 or 3 is both okay on pcb_v2/pcb_v2b. It's just different pin spacing.
> Caps and resistors form a signal filter which may or may not be needed.
> I'd suggest "the red" level shifter. It seems to be more successful but as with the filters you may have to experiment a little if you have a picky pump. Search for 8 channel bi directional level shifter on eBay.

Hi there, thank you for your prompt reply!

Is this what you mean by the red level shifter?

https://www.ebay.co.uk/itm/122940101058?mkcid=16&mkevt=1&mkrid=711-127632-2357-0&ssspo=LH2mHz_kRky&sssrc=4429486&ssuid=&var=&widget_ver=artemis&media=COPY

**Comment by visualapproach** [comment]
Oh sorry I see now you did search on eBay LOL. Well, you should see some red ones there too.

**Comment by owenashurst** [comment]
> Oh sorry I see now you did search on eBay LOL. Well, you should see some red ones there too.

Just noticed you replied about the shifter too lol. Cool I'll order that one I linked if you think that's good?

I'll return this nodemcu v1 board then and grab a v3.

**Reply by visualapproach** [reply]
Sorry for delayed answer. Busy times. Yeah that LLC seems to be right
